105/Hw/Hw/HomeViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  HomeViewController.swift
+//  Hw
+//
+//  Created by Максим Тарасов on 07.10.2022.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    @IBOutlet weak var animalImageButton: UIButton!
+    @IBOutlet weak var browseButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        animalImageButton.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(animalImageButton)
+        browseButton.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(browseButton)
+        
+        NSLayoutConstraint.activate([
+            NSLayoutConstraint(item: animalImageButton, attribute: .centerY,
+                               relatedBy: .equal, toItem: view,
+                               attribute: .centerY, multiplier: 1,
+                               constant: 200),
```

Comment:
1. Давай использовать anchors, они более читаемы.
3. Давай переносить параметры после скобки, чтобы отступы были такими:
```suggestion
            NSLayoutConstraint(
                item: animalImageButton, attribute: .centerY,
                relatedBy: .equal, toItem: view,
                attribute: .centerY, multiplier: 1,
                constant: 200
            ),
```

Reply:
3. Ворнинги. Их не стоит игнорировать

> Coercion of implicitly unwrappable value of type 'UITextField?' to 'Any' does not unwrap optional

По-хорошему нужно создавать вьюшки в коде, а не storyboard, либо чтобы заглушить ворнинги нужно хотя бы сделать guard let с вьюшками.